frontend/src/components/GoesData/AnimationStudioTab/StudioSettings.tsx

```
@@ -0,0 +1,126 @@
+        <div>
+          <label htmlFor="anim-fps-fps" className="block text-xs text-gray-400 dark:text-slate-500 mb-1">FPS: {fps}</label>
+          <input id="anim-fps-fps" type="range" min={1} max={30} value={fps} onChange={(e) => setFps(Number(e.target.value))}
+            className="w-full accent-primary" />
+        </div>
+
+        <div>
+          <label htmlFor="anim-format" className="block text-xs text-gray-400 dark:text-slate-500 mb-1">Format</label>
+          <div className="flex gap-2">
+            {(['mp4', 'gif'] as const).map((f) => (
+              <button key={f} onClick={() => setFormat(f)}
+                className={`px-4 py-1.5 text-sm rounded-lg ${format === f ? 'bg-primary text-gray-900 dark:text-white' : 'bg-gray-100 dark:bg-slate-800 text-gray-500 dark:text-slate-400'}`}>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)